server/docker-base/Dockerfile

```diff
@@ -377,14 +377,14 @@ RUN \
 # PS4 SDK
 #
 
-ENV PS4_SDK_10_000 ${PLATFORMSDK_DIR}/ps4-sdk-10.000
+ENV PS4_SDK_10  ${PLATFORMSDK_DIR}/ps4-sdk-10.500
```

Comment:
By not using a minor version, we can update the sdk without changes to the dmSDK